Resources/Prototypes/Entities/Markers/Spawners/Random/posters.yml

```
@@ -92,6 +92,8 @@
       - PosterContrabandInterdyne
       - PosterContrabandWaffleCorp
       - PosterContrabandMissingSpacepen
+      - PosterContrabandYakub # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)